docs/root/modules/kubernetes/schema.md

```diff
@@ -154,3 +163,175 @@ Representation of a [Kubernetes Secret.](https://kubernetes.io/docs/concepts/con
     ```
     (:KubernetesNamespace)-[:CONTAINS]->(:KubernetesSecret)
     ```
+
+### KubernetesServiceAccount
+Representation of a [Kubernetes ServiceAccount.](https://kubernetes.io/docs/concepts/security/service-accounts/)
+
+| Field | Description |
+|-------|-------------|
+| id | Identifier for the ServiceAccount derived from namespace and name (e.g. `default/my-service-account`) |
```

Comment:
update to mention cluster, applies to other new Kube objects documented in this PR